test/generator/print_helpers_test.dart

```diff
@@ -47,13 +47,13 @@ void main() {
       final str = specToString(enumDefinitionToSpec(definition));
 
       expect(str, '''enum Name {
-  @JsonValue("Option")
+  @JsonValue('Option')
```

Comment:
What is the cause of this replacements from  `"` to `'`?

Reply:
Dart best practices prefer the use of single quotes where they don't require additional escapes: https://dart-lang.github.io/linter/lints/prefer_single_quotes.html

Reply:
Dart best practices prefer the use of single quotes when there aren't additional escapes required: https://dart-lang.github.io/linter/lints/prefer_single_quotes.html